src/tim/vx/ops/avg_pool_test.cc

```diff
@@ -27,6 +27,7 @@
 #include <iostream>
 #include "gtest/gtest.h"
 #include "test_utils.h"
+#include "tim/transform/layout_inference.h"
```

Comment:
not include layout inference here. We have layout inference unit test.

Reply:
done